tests/travis.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
So convention is to create a directory named `.travis` and have this as `run_tests.sh` or something similar. I'd prefer to follow that convention in this case, if you wouldn't mind. :) If you agree I can fix this up and push it to master.